src/helpers/transitionEnd.js

```diff
@@ -1,20 +1,9 @@
 const transitionEnd = () => {
-	const el = document.createElement('div');
-
-	const transEndEventNames = {
-		WebkitTransition: 'webkitTransitionEnd',
-		MozTransition: 'transitionend',
-		OTransition: 'oTransitionEnd otransitionend',
-		transition: 'transitionend'
-	};
-
-	for (let name in transEndEventNames) {
-		if (el.style[name] !== undefined) {
-			return transEndEventNames[name];
-		}
+	if (window.ontransitionend === undefined && window.onwebkittransitionend !== undefined) {
+		return 'webkitTransitionEnd'
+	} else {
+		return 'transitionend'
```

Comment:
Not sure why is this change needed. Are the values different nowadays?

Reply:
I thought the `opera` part looked dated, then just looked at how Vue currently handles transitions as it's baked into the core and works really well, and that's how they do it 🤠 We could revert this part to how it's done currently but this one is less code. However the opera part is not necessary anymore since they switched to Chromium as well. Want to revert it?

Reply:
Nah, just curious, let's roll with it!